a10_octavia/tests/unit/controller/worker/tasks/test_network_tasks.py

```diff
@@ -14,36 +14,74 @@
 
 
 import mock
+from unittest.mock import patch
 
+from octavia.common import data_models as o_data_models
 from octavia.tests.common import constants as t_constants
-import octavia.tests.unit.base as base
+from octavia.tests.unit import base
 
 from a10_octavia.controller.worker.tasks import a10_network_tasks
 from a10_octavia.network import data_models
 from a10_octavia.network.data_models import Subport
 from a10_octavia.tests.common import a10constants as a10_test_constants
 
-SUBPORT = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID)
+SUBPORT1 = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID1)
+SUBPORT2 = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID2)
+
+VIP = o_data_models.Vip(port_id=t_constants.MOCK_PORT_ID,
+                        subnet_id=t_constants.MOCK_SUBNET_ID,
+                        qos_policy_id=t_constants.MOCK_QOS_POLICY_ID1)
+LB = o_data_models.LoadBalancer(vip=VIP)
+PARENT_PORT = data_models.ParentPort(trunk_id=a10_test_constants.TRUNK_ID)
+EMPTY_PARENT_PORT = data_models.ParentPort()
+DELETE_SUBPORTS = {t_constants.MOCK_AMP_ID1: [SUBPORT1]}
```

Comment:
These variables should be defined within the methods they are testing. Up here we are solely defining the constant data models being used to fill out the example data.